spec/services/reports/children_served_report_service_spec.rb

```diff
@@ -108,6 +108,8 @@
                                   })
     end
   end
+
+  # TODO: figure tis out
   describe "#disposable_diapers_from_kits_total" do
     it "calculates the number of disposable diapers that have been distributed within kits" do
       toddler_disposable_kit = create(:kit, :with_item, organization: organization)
```

Comment:
I can't figure this test out. Not quite sure what it is doing and with my changes its giving me 400 at the end. Could use an assist.